README.md

```diff
@@ -1,22 +1,174 @@
-## Example command
+[![Build](https://github.com/canonical/chisel/actions/workflows/build.yml/badge.svg)](https://github.com/canonical/chisel/actions/workflows/build.yml)
+[![Tests](https://github.com/canonical/chisel/actions/workflows/tests.yaml/badge.svg)](https://github.com/canonical/chisel/actions/workflows/tests.yaml)
 
+# Chisel
+
+**Chisel** is a software tool for carving and cutting Debian packages!
+
+It is built on the idea of **Package Slices** - minimal, complimentary and
```

Comment:
Package Slices should probably be lowercased as it's not a proper noun. The emphasis is nice, though.